Sources/Public/CalendarView.swift

```diff
@@ -567,16 +567,6 @@ public final class CalendarView: UIView {
     } else {
       view.selectionHandler = nil
     }
-
-    // Only enable user interaction for day items if a `daySelectionHandler` is set
-    if
-      case .layoutItemType(.day) = visibleItem.itemType,
-      daySelectionHandler != nil
-    {
-      view.isUserInteractionEnabled = true
-    } else {
-      view.isUserInteractionEnabled = false
-    }
```

Comment:
If it's not a day view, then the selection handler won't get invoked since it's never set to anything besides `nil`. See https://github.com/airbnb/HorizonCalendar/pull/59/files#diff-cb95d9bcbe378d78650b19323f50a949L565